pycm/pycm_param.py

```diff
@@ -38,6 +40,8 @@
 
 COMPARE_WEIGHT_ERROR = "The weight type must be dictionary and also must be specified for all of the classes."
 
+COMBINE_TYPE_ERROR = "The input type is considered to be pycm.ConfusionMatrix object but it's not!"
```

Comment:
The input type is supposed to be pycm.ConfusionMatrix object but it's not!

Reply:
This error message is exactly like `COMPARE_TYPE_ERROR`

Reply:
What's your opinion about changing both of them?

Reply:
Done!
Please take a look at https://github.com/sepandhaghighi/pycm/pull/344/commits/c9edcb62e5aae931d5b7231540857b64280083e2 and https://github.com/sepandhaghighi/pycm/pull/344/commits/2e4c543a6806dde36c3f709933f0f1fee2e0eefb

`COMPARE_FORMAT_ERROR` , `COMPARE_TYPE_ERROR`, `COMBINE_TYPE_ERROR`, `CI_FORMAT_ERROR` updated

Reply:
Also, take a look at #345